commands/utils.py

```diff
@@ -67,6 +67,8 @@ async def deploy(
     production: bool,
     description: str,
     check_constraints: bool,
+    exclude_check_run_name: list[str],
```

Comment:
I think the types here do not match. You use `tuple`s for the default values, but the type here is `list`. You might want to use e.g. `typing.Sequence` instead or stick to one of `tuple` or `list`.

Reply:
Done.